wallet/lib/pages/purchase_item/purchase_item_screen.dart

```diff
@@ -190,6 +209,77 @@ class _PurchaseItemContentState extends State<PurchaseItemContent> {
       }
     }
   }
+
+  void _onVerticalDragStart(DragStartDetails details) {
```

Comment:
We should have one widget that encompasses this gestures logic and that we can use on purchase and owner